Description: Fix stderr leakage from grub-probe in is_path_readable_by_grub
Author: Colin Watson <cjwatson@ubuntu.com>
Origin: upstream, http://bazaar.launchpad.net/~vcs-imports/grub/grub2-bzr/revision/4590
Forwarded: not-needed
Applied-Upstream: http://bazaar.launchpad.net/~vcs-imports/grub/grub2-bzr/revision/4590
Last-Update: 2012-09-26

Index: b/util/grub-mkconfig_lib.in
===================================================================
--- a/util/grub-mkconfig_lib.in
+++ b/util/grub-mkconfig_lib.in
@@ -65,7 +65,7 @@
 
   # ... or if we can't figure out the abstraction module, for example if
   # memberlist fails on an LVM volume group.
-  if abstractions="`"${grub_probe}" -t abstraction "$path"`" 2>&1 ; then 
+  if abstractions="`"${grub_probe}" -t abstraction "$path"`" 2> /dev/null ; then 
       :
   else
     return 1
